code/controllers/subsystem/persistence.dm

```diff
@@ -261,6 +309,34 @@ SUBSYSTEM_DEF(persistence)
 		data["placer_key"] = T.placer_key
 		saved_trophies += list(data)
 
+/datum/controller/subsystem/persistence/proc/SaveFrame(obj/structure/sign/picture_frame/persist/F)
```

Comment:
Something has gone wrong with your art persistence please disable all this code and try a second test merge to confirm this please. It appears to be crashing everyone including the latest beta people.